Gemfile

```diff
@@ -38,3 +38,4 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+gem 'rubocop', require: false
```

Comment:
It would be better if you provide gems with versions. Go on this [website](https://rubygems.org/gems/rubocop) and search for a gem, for example
```
gem 'rubocop', '~> 1.18', '>= 1.18.4'
```

It is important because if someone clones your repo he would have the same gems versions as you do. We will clone your repositories to play with searching for tough bugs :)